src/coreclr/tools/aot/ILCompiler.Compiler/ILCompiler.Compiler.csproj

```diff
@@ -20,6 +20,11 @@
     <ProjectReference Include="..\ILCompiler.DependencyAnalysisFramework\ILCompiler.DependencyAnalysisFramework.csproj" />
     <ProjectReference Include="..\ILCompiler.MetadataTransform\ILCompiler.MetadataTransform.csproj" />
     <ProjectReference Include="..\ILCompiler.TypeSystem\ILCompiler.TypeSystem.csproj" />
+
+    <PackageReference Include="StaticCs" Version="$(StaticCsVersion)">
```

Comment:
I think this will be a problem for source build.

It looks like this is just some analyzer with a custom attribute? I would be fine not running the analyzer in source build (or ever) and we can fake the custom attribute so that we don't have to make acrobatics in the shared files.

Reply:
@agocke - could you please comment on this? I basically just copied this from linker, I don't know the details behind it.

Reply:
Yup, it's an analyzer and a source file. I think we can condition it out in source build.